File: day3/main.go
package main

import (
	"fmt"
)


func promptPassword() string {
	var password string
	fmt.Print("Enter Password: ")
	fmt.Scan(&password)
	return password
}


func main() {
	secretPassword := "secret123"
	var password string
	for {
		password = promptPassword()
		if password == secretPassword {
			fmt.Println("Access Granted!")
			break
		} else {
			fmt.Println("Incorrect, try again.")
		}
	}
}

File: day3/main.py
def prompt_password() -> str:
    password = input("Enter Password: ")
    return password


def main():
    secret_password = "secret123"
    password = ""
    while password != secret_password:
        password = prompt_password()
        if password != secret_password:
            print("Incorrect, try again.")
        else:
            print("Access Granted!")
    
    


if __name__ == "__main__":
    main()